docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -1,387 +1,299 @@
+
+## Hardware Requirements
+
+You need an an Espressif ESP32 microcontroller to use the micro-RDK.
+Viam recommends purchasing the ESP32 with a development board: see [development kit options](https://www.espressif.com/en/products/devkits).
+
+Minimal configuration: 384kB Ram 4MB flash
+Recommended configuration: 384kB Ram + 8MB SPIRAM + 4MB Flash
+
+## Software Requirements
+
+The micro-RDK is written in Rust.
+
+We recommend using Viam's [Canon utility](#using-canon) to build the `micro-rdk` server, as the utility downloads and installs a Docker development environment automatically.
+You can set up the development environment manually by following [these instructions](#set-up-development-environment-manually).
+
+To use the micro-RDK with your ESP32 board, follow these steps:
+
+- [Install prerequisites](#install-prerequisites)
+
+If using the Docker environment with Canon:
+
+- [Use Canon](#using-canon)
+
+If setting up the environment yourself (Optional):
+
+- [Set up development environment manually](#set-up-development-environment-manually)
+
+The following instructions cover installation for macOS and Linux machines.
+
+### Install prerequisites
+
+To build the micro-rdk, you must install the following software on your machine:
+
+#### Install dependencies
+
+{{< tabs >}}
+{{% tab name="Linux (Ubuntu)" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo apt-get install git libssl-dev dfu-util libusb-1.0-0 libudev-dev
+```
+
+{{% /tab %}}
+{{% tab name="macOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew install dfu-util
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+#### Install Rust
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+See [Rust](https://www.rust-lang.org/tools/install) for more information and other installation methods.
+
+#### Install `cargo-generate` with `cargo`
+
+ `cargo` installs automatically when downloading Rust with rustup.
+
+Run the following command to install `cargo-generate`:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install cargo-generate
+```
+
+#### Install `espflash`
+
+Run the following command to install `espflash`
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+cargo install espflash
+```
+
+### Using Canon
+
+Canon is a CLI utility for managing a Docker based canonical environment.
+See [GitHub](https://github.com/viamrobotics/canon) for more information.
+It needs  a working installation of docker, follow [Docker Installation](https://docs.docker.com/engine/install/) to install Docker. If running linux make sure to go throught the post installation [steps](https://docs.docker.com/engine/install/linux-postinstall/)
+
+#### Homebrew
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+brew install viamrobotics/brews/canon
+```
+
+#### Direct Install
```

Comment:
Do I need to do both homebrew and direct install - we need more explanation /guidance here

Reply:
see above